odoo/addons/base/models/res_company.py

```diff
@@ -194,8 +194,9 @@ def _compute_uninstalled_l10n_module_ids(self):
         self.env.cr.execute("""
             SELECT country.id,
                    ARRAY_AGG(module.id)
```

Comment:
Won't you have multiple times the same `module.id` in the aggregate now?

Reply:
if you have multiple times the same module_id in module_country table for a given country_id, yes.